### PR TITLE
More in Anvil

### PR DESCRIPTION
**Pull request by NycuRO**

### CODE REVIEW

**Comment by NycuRO** [comment]
@Creeperface01 It's correctly?

File: src/main/java/cn/nukkit/inventory/AnvilInventory.java [hunk 1]
```diff
@@ -44,9 +44,9 @@ public boolean onRename(Player player, Item resultItem) {
         }
 
         if (local.getId() != 0 && second.getId() == 0) { //only rename
+            player.getInventory().addItem(local);
```
Comment (NycuRO) [inline src/main/java/cn/nukkit/inventory/AnvilInventory.java]: @Creeperface01 #Blame_Creeper. First is this for fix bug of duplicate item ;/

File: src/main/java/cn/nukkit/item/Item.java [hunk 2]
```diff
@@ -1342,6 +1342,33 @@ public static Item get(int id, Integer meta, int count, byte[] tags) {
...
+    }
+
+    public static Item send(int id, Integer meta) {
+        return send(id, meta, 1);
+    }
+
+    public static Item send(int id, Integer meta, int count) {
+        return send(id, meta, count, new byte[0]);
+    }
+
```
Comment (Pub4Game) [inline src/main/java/cn/nukkit/item/Item.java]: what? it very impractical

**Comment by Pub4Game** [comment]
You broke the renaming of the item because it is issued before setCustomName()

**Comment by Creeperface01** [comment]
I or somebody else will improve anvil soon

**Pull request closed without merging**